pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -238,6 +238,14 @@ let
       sed -i -e '/libpci_loader.*Load/s!"\(libpci\.so\)!"${pciutils}/lib/\1!' \
         gpu/config/gpu_info_collector_linux.cc
 
+      sed -i                                -e 's,CHECK(!check || library);,CHECK(!check || library) << library_name;,'            \
```

Comment:
So it is not enough to add those library paths to the RPATH of chromium?
If so why would this only be caused in some window managers?
RPATH is in the elf binary itself and should not be dependent on environment variables.

Reply:
>> So it is not enough to add those library paths to the RPATH of chromium?

Indeed.
adding them to `$LD_LIBRARY_PATH` does the trick

>> If so why would this only be caused in some window managers?

Because they do not bring `gtk3` and `gdk-pixbuf`.

>> RPATH is in the elf binary itself and should not be dependent on environment variables.

Not always:
> (ELF only) If the executable file for the calling program contains a DT_RPATH tag, and does not contain a DT_RUNPATH tag, then the directories listed in the DT_RPATH tag are searched. (https://linux.die.net/man/3/dlopen)

In fact, I get crash on this `CHECK`:
https://github.com/chromium/chromium/blob/672d7d265cc72a8c056cdc7f81fe9c43d01b988d/ui/gtk/gtk_compat.cc#L43-L45

Reply:
`gtk3` and `gdk-pixbuf` is not even in RPATH of `$out/bin/chromium`
Perhaps, `patchelf` will fix it too